ceph-releases/ALL/ubuntu/daemon-base/__EXTRA_POSTINSTALL_CLEANUP__

```diff
@@ -0,0 +1 @@
+dpkg -s __CEPH_BASE_PACKAGES__
```

Comment:
@ErwanAliasr1 : As an aside, we forgot to change these to `apt-cache show` (I think `show` is the right command) when we made the point release update. Maybe you can make a new PR for this?

Reply:
`apt-cache show` is correct


```
root@2d051cc01bd6:/# apt-cache show sed=4.2.2-7
Package: sed
Essential: yes
Status: install ok installed
Priority: required
Section: utils
Installed-Size: 304
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Architecture: amd64
Multi-Arch: foreign
Version: 4.2.2-7
Depends: dpkg (>= 1.15.4) | install-info
Pre-Depends: libc6 (>= 2.14), libselinux1 (>= 1.32)
Description: The GNU sed stream editor
 sed reads the specified files or the standard input if no
 files are specified, makes editing changes according to a
 list of commands, and writes the results to the standard
 output.
Description-md5: 67b5a614216e15a54b09cad62d5d5afc
Original-Maintainer: Clint Adams <clint@debian.org>
Homepage: http://www.gnu.org/software/sed/

root@2d051cc01bd6:/# apt-cache show sed=4.2.3-7
E: Version '4.2.3-7' for 'sed' was not found
E: No packages found
```

Reply:
And when version is not found, the return code is `100`, so it does error. =)

Reply:
I added a very simple commit to handle that and put you as co-author.